src/edu/csus/ecs/pc2/core/strategies/MaxSubmissionsPerMinuteStrategy.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.strategies;
+
+import edu.csus.ecs.pc2.core.IThrottleStrategy;
+import edu.csus.ecs.pc2.core.model.ClientId;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.Run;
+
+/**
+ * This class implements an {@link IThrottleStrategy} which rejects runs when the submitting team
+ * has submitted more than "N" runs in the last minute.  The value of N is set when the class is instantiated.
+ * 
+ * @author John Clevenger
+ *
+ */
+public class MaxSubmissionsPerMinuteStrategy implements IThrottleStrategy {
+    
+    public static int DEFAULT_MAX_SUBMISSIONS_PER_MINUTE = 10 ;
+
+    private  IInternalContest contest;
+    private int maxPerMinute;
+    
+    /**
+     * Construct a strategy with a default limit on submissions per minute.
+     * 
+     * @param inContest the {@link IInternalContest} with which this strategy is associated.
+     */
+    public MaxSubmissionsPerMinuteStrategy (IInternalContest inContest) {
+        this.contest = inContest ;
+        this.maxPerMinute = DEFAULT_MAX_SUBMISSIONS_PER_MINUTE ;
+    }
+
+    /**
+     * Construct a strategy with a caller-specified limit on submissions per minute.
+     * 
+     * @param inContest the {@link IInternalContest} with which this strategy is associated.
+     */
+    public MaxSubmissionsPerMinuteStrategy (IInternalContest inContest, int maxAllowed) {
+        this.contest = inContest ;
+        this.maxPerMinute = maxAllowed ;
+    }
+
+    @Override
+    /**
+     * Rejects the specified run if the submitting team has already submitted the maximum allowed number of runs in the past minute. 
+     */
+    public boolean accept(Run run) {
+        return numSubmittedInTheLastMinuteExceedsMaximum(contest, run);
```

Comment:
I think this should be:
`return !numSubmittedInTheLastMinuteExceedsMaximum(contest, run);`
Basically, I can't make *any* submissions ATM.

Reply:
Fixed in most recent push.